server/routes/visitJourney/visitJourneyUtils.ts

```diff
@@ -0,0 +1,6 @@
+// return URL prefix for either visit booking or visit update journey
+const getUrlPrefix = (isUpdate: boolean, previousVisitReference: string) => {
```

Comment:
Was in two minds whether or not to create yet another 'utils' file but figured there's likely to be other things here to support these 'visitJourney' classes - and these various utils files probably need sorting out at some point anyway...